src/java/org/apache/cassandra/io/sstable/metadata/StatsMetadata.java

```diff
@@ -267,7 +271,24 @@ public int serializedSize(Version version, StatsMetadata component) throws IOExc
             size += EstimatedHistogram.serializer.serializedSize(component.estimatedPartitionSize);
             size += EstimatedHistogram.serializer.serializedSize(component.estimatedCellPerPartitionCount);
             size += CommitLogPosition.serializer.serializedSize(component.commitLogIntervals.upperBound().orElse(CommitLogPosition.NONE));
-            size += 8 + 8 + 4 + 4 + 4 + 4 + 8 + 8; // mix/max timestamp(long), min/maxLocalDeletionTime(int), min/max TTL, compressionRatio(double), repairedAt (long)
+            size += 8 + 8; // mix/max timestamp(long)
+            if (version.hasLongLocalDeletionTime())
```

Comment:
We have two identical if-else blocks, maybe we can merge them.

Reply:
I'll resolve but feel free to reopen if you disagree ofc.

Reply:
Works for me. By the way, I think reviewers cannot reopen conversations once they are marked as resolved.

Reply:
Ok but I see you can comment and I'll be happy to reopen if I ever close sthg and somebody disagrees.